client/src/index.css

```diff
@@ -0,0 +1,19 @@
+body {
```

Comment:
a good way to start a css is

```
* {
padding: 0;
margin: 0;
}
```
then we prevent the default behavior of a browser to add margin and padding for certain elements

Reply:
cool, I used it, thanks!